source/_posts/2017/2017-06-03-thank-you-david.md

```diff
@@ -0,0 +1,90 @@
+
+A great friend of mine [Honza Černý](http://honzacerny.com/) lend me a book *[Steal Like An Artist](http://austinkleon.com/steal/)* few months ago. I'm very grateful for that, because I don't have much art-people in my social circle. You may think, it's about art (so did I), but it's **more about relationships, sharing, caring, showing your vulnerability and cooperation**. And open-source software!
+
+One of lessons that lead me to this was...
+
+## Write Letters to Your Heroes
+ 
+> A lot of times when we write fan letters we’re looking for a blessing or an affirmation. 
+> But if you truly love somebody's work, you shouldn't need a response from them.
+> Write a blog post about someone's work that you admire and link to their site. 
+> Make something and dedicate it to your hero. 
+> Answer a question they've asked, solve a problem for them, or improve on their work and share it online.
+
+After reading this, I had my hero in mind, whom I'd like to write. I also shared this though with [Honza Mikeš](https://github.com/Lexinek), who answered me by showing a beautiful thank you email he wrote.
+
+So here is my turn:
+
+
+## Thank you David
+
+I'd like to really thank you for 3 things you gave me.
+
+## Thank you for my job 
+ 
+I'm very grateful you've created Nette and take care of it for last 10 years. I could work on many interesting web projects, earn a living from it and learn best practise in open-source. I've maintained like dozens of packages for a few years and **I could barely remotely imagine, what it takes to maintain one project mostly by yourself. I envy you your huge persistence and determination**.
+
+**Thanks to you, I could join 3 things I love the most in the worlds - people, open-source and education** - and get to teaching, learning and growing a community. I must say it was a relief, because when I was at university, I felt like I could do only programming or only work with people in psychology field. 
+
+### It's easy to be leader, or not?
+
+I remember one moment, when I got much respect for you. Once we had a social training group, when me and 15 more schoolmates went to a cottage near Brno to experience social dynamics in games. There was a boat. And every person on this boat had it's role. We discussed in a group who would be sailor, who would be a navigator and who would be a captiain.
+
+I though it must be easy to be a captain (or Nette creator), because you can tell people what to do and you can decide mostly by yourself. Oh how hard lesson was about to come. We sailed over the sea, sun was shining, we had enough food and I was looking at my shiny boat with huge happy crazy. And life was great. So far. 
+
+Suddenly, navigator noticed trees on a horizon - there was a land. Feeling of joy and excitement washed over the crew. And life was great.
+ 
+Then, a steersman started to feeling sick and had to lay down. How do we get without steersman to the land? I quickly find very skillful replacement. And life was good.
```

Comment:
kde se tam vzal steersman najednou? :-D 
za "Then" bych nedělala čárku
had to lie down
How do we get to the land without the steersman?
I quickly found